### PR TITLE
Styling and Icons

### PR DESCRIPTION
**Pull request by edwardstamps**

### CODE REVIEW

File: web/admin/src/ContentDetailsEditor.js [hunk 1]
```diff
@@ -19,27 +18,27 @@ export default class ContentDetailsEditor extends PureComponent {
...
           </div>
           <div className="homeBox">
-            <h2>Add Content</h2>
+            <h2 className="contentTitle">Add Content</h2>
             <TextEditor content={c} prop="url" title="URL" regex={/^https?:\/\/[^/]/} validationMessage="The URL must begin with 'https://' or'http://'" placeholder="http://www.acme.com" onUpdate={onUpdate} />
           </div>
         </div>
       case 'survey': return <div>
-          <h2>Select a global survey</h2>
-          <SelectEditor size={6} content={c} prop="surveyId" title="Survey" onUpdate={onUpdate} options={surveys} />
```
Comment (AdamLiechty) [inline web/admin/src/ContentDetailsEditor.js]: Don't we want to keep the size=6?

File: web/admin/src/editors.js [hunk 2]
```diff
@@ -46,19 +46,19 @@ export class SelectEditor extends PureComponent {
...
   render() {
-    const {content, options, prop, size, title} = this.props
+    const {content, options, prop, size} = this.props
 
     return (
       <label className="select-editor">
-        <div className="select-editor__title">{title}</div>
         <select
-          size={size || 0}
+          // size={size || 0}
```
Comment (AdamLiechty) [inline web/admin/src/editors.js]: why is this removed?

Comment (edwardstamps) [reply web/admin/src/editors.js]: The size props was not being passed to this page temporarily took it out

**Comment by AdamLiechty** [comment]
replaced by #23
